frontend/src/__tests__/api.test.js

```diff
@@ -96,95 +95,230 @@ function sessionsTests(api = { apiGET, apiPOST }) {
-        expect(resp1).toMatchObject({ status: "success" });
-        const sessionId = resp1.payload.id;
 
         // grab the position_templates of the new session. They may have
         // pre-populated.
-        const resp2 = await apiGET(`/sessions/${sessionId}/position_templates`);
-        expect(resp2).toMatchObject({ status: "success" });
+        const resp1 = await apiGET(
+            `/sessions/${session.id}/position_templates`
+        );
+        expect(resp1).toMatchObject({ status: "success" });
         checkPropTypes(
             PropTypes.arrayOf(offerTemplatePropTypes),
-            resp2.payload
+            resp1.payload
         );
 
         // add the new offer template
-        const resp3 = await apiPOST(
-            `/sessions/${sessionId}/add_position_template`,
-            newTemplateData
+        const resp2 = await apiPOST(
+            `/sessions/${session.id}/add_position_template`,
+            newTemplateData1
         );
-        expect(resp3).toMatchObject({ status: "success" });
+        expect(resp2).toMatchObject({ status: "success" });
         checkPropTypes(
             PropTypes.arrayOf(offerTemplatePropTypes),
-            resp3.payload
+            resp2.payload
         );
-        expect(resp3.payload).toContainObject(newTemplateData);
+        expect(resp2.payload).toContainObject(newTemplateData1);
 
         // another one -- it should contain both of our added templates
-        const resp4 = await apiPOST(
-            `/sessions/${sessionId}/add_position_template`,
+        const resp3 = await apiPOST(
+            `/sessions/${session.id}/add_position_template`,
             newTemplateData2
         );
-        expect(resp4.payload).toContainObject(newTemplateData);
-        expect(resp4.payload).toContainObject(newTemplateData2);
+        expect(resp3.payload).toContainObject(newTemplateData1);
+        expect(resp3.payload).toContainObject(newTemplateData2);
 
         // fetching the templates again should give us the same list
-        const resp5 = await apiGET(`/sessions/${sessionId}/position_templates`);
-        expect(resp5.payload).toEqual(resp4.payload);
+        const resp4 = await apiGET(
+            `/sessions/${session.id}/position_templates`
+        );
+        expect(resp4.payload).toEqual(resp3.payload);
+    });
+    it("update a template", async () => {
+        const newTemplateDataToUpdate = {
+            offer_template: "this_is_a_test_template_to_be_updated.html",
+            position_type: "Standard"
+        };
+
+        // add the new test template
+        const resp1 = await apiPOST(
+            `/sessions/${session.id}/add_position_template`,
+            newTemplateDataToUpdate
+        );
+        expect(resp1).toMatchObject({ status: "success" });
+        checkPropTypes(
+            PropTypes.arrayOf(offerTemplatePropTypes),
+            resp1.payload
+        );
+        expect(resp1.payload).toContainObject(newTemplateDataToUpdate);
```

Comment:
We already checked this stuff in the previous test. We don't need to do it again. (add a comment though, that `add_position_template` was already tested.)